experimental/AlgebraicStatistics/src/DiscreteGraphicalModels.jl

```diff
@@ -0,0 +1,86 @@
+function bron_kerbosch(G, R, P, X)
+
+    cliques = []
+
+    if length(P) == 0 && length(X) == 0
+        
+        push!(cliques, R)
+    end
+
+    Q = P
+    Y = X
+
+    for v in Q
+
+        new_R = vcat(R, v);
```

Comment:
No need for those semicolons :-)
```suggestion
        new_R = vcat(R, v)
```